README.md

```diff
@@ -37,6 +37,7 @@
 - [Paw](https://luckymarmot.com/paw) - The ultimate REST client for Mac.
 - [PSequel](http://www.psequel.com/) - A PostgreSQL GUI tool.
 - [Sequel Pro](http://www.sequelpro.com/) - A MySQL database manager.
+- [SourceTree](https://www.sourcetreeapp.com/) - A free Git & Mercurial client for Mac & Windows.
```

Comment:
Yeah I thought so first too, but it might be worth mentioning it's cross-platform (well, Mac & Windows) for users who wants to use the same tools on both platforms?